cmd/bundle/run.go

```diff
@@ -19,7 +19,27 @@ func newRunCommand() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "run [flags] KEY",
 		Short: "Run a resource (e.g. a job or a pipeline)",
-		Args:  root.MaximumNArgs(1),
+		Long: `Run a resource (e.g. a job or a pipeline)
```

Comment:
In DECO-2549 I suggested "Run the job or pipeline identified by the KEY.", mostly because "running a resource" sounds weird to me and if it's just those two, then why not just say that. But if it applies to all resources then maybe "Start a run of a resource"?  

Also, minor, but if there has to be a parenthetical, this is friendlier: "(for example, a job or a pipeline)"

Reply:
Thank you, applied!